### DISCUSSION TITLE
setting bruno on WSL

### DISCUSSION THREAD

**Discussion by sanjai0py** [usebruno/bruno#1201] [General]
while setting the project up on WSL people might face errors like `error while loading shared libraries: libgbm.so.1: cannot open shared object file: No`, `error while loading shared libraries: libgtk-3.so.0: cannot open shared object file:`, ` error while loading shared libraries: libcups.so.2: cannot open shared object file: No`, `error while loading shared libraries: libatk-bridge-2.0.so.0: cannot open shared object file: No such file or directory`

solution for this error is to install the lib essentials for Electron js by following [this instructions](https://www.electronjs.org/docs/latest/development/build-instructions-linux)

**Comment by Ayo-Awe** [comment]
I keep getting a segfault every time I try running bruno in wsl 
I'm running on Ubuntu 22.04.2 LTS

<img width="613" alt="image" src="https://github.com/usebruno/bruno/assets/82258944/8e6e945a-6961-4a99-a3cd-feceff34405a">

**Reply by sanjai0py** [reply]
Can you please mention how did you run Bruno? Did you built it from the source?

**Reply by Ayo-Awe** [reply]
Hi @sanjai0py, I didn't build from source, I downloaded the binaries and installed them using apt. I also tried installing using snap, but I got the same error.

**Reply by sanjai0py** [reply]
can you please mention the version of WSL you use?

**Reply by Ayo-Awe** [reply]
WSL version: 2.0.9.0
Kernel version: 5.15.133.1-1
WSLg version: 1.0.59
MSRDC version: 1.2.4677
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.2715

@sanjai0py anything else?

**Reply by sanjai0py** [reply]
just tested on WSL 2 and it works fine for me. please try `snap remove bruno`, `wsl --shutdown` and try reinstalling `snap install bruno` and run it with `sudo snap run bruno` always update your `apt packages` before installing

**Reply by Ayo-Awe** [reply]
Thanks, it worked

**Reply by sanjai0py** [reply]
that's great!
